src/main/java/com/redislabs/redisai/Script.java

```diff
@@ -0,0 +1,140 @@
+package com.redislabs.redisai;
+
+import com.redislabs.redisai.exceptions.JRedisAIRunTimeException;
+import redis.clients.jedis.util.SafeEncoder;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Script {
+
+    /**
+     * the device that will execute the model. can be of CPU or GPU
+     */
+    private Device device;
+
+
+    /**
+     * a string containing TorchScript source code
+     */
+    private String source;
+
+    /**
+     * tag is an optional string for tagging the model such as a version number or any arbitrary identifier
+     */
+    private String tag;
+
+
+    /**
+     * the device that will execute the model. can be of CPU or GPU
+     *
+     * @param device
+     * @param source a string containing TorchScript source code
+     */
+    public Script(Device device, String source) {
+        this.device = device;
+        this.source = source;
+        this.tag = "";
```

Comment:
@filipecosta90 are you sure that empty string is the right default?

Reply:
solved on the new commit